internal/local/features.go

```diff
@@ -68,58 +68,67 @@ func (service *ProvisioningService) SetAMTFeatures() error {
 		return utils.AMTFeaturesConfigurationFailed
 	}
 
-	//Get OptInService
-	getOptInServiceResponse, err := service.interfacedWsmanMessage.GetIpsOptInService()
-	if err != nil {
-		log.Error("Error while getting the OptIn Service: ", err)
-
-		return utils.AMTFeaturesConfigurationFailed
-	}
+	if service.flags.ControlMode == 2 {
```

Comment:
do we have a const available for control modes to check against instead of "2"

Reply:
No, we don't currently have a constant for this. I'll create a separate refactoring PR to introduce proper constants for activation modes, eliminating these numbers.